prisma/schema.prisma

```diff
@@ -37,6 +37,8 @@ model Group {
   id          String  @id @default(uuid())
   name        String
   description String  @db.Text
+  imgUrl      String?
+  imgAlt      String?
```

Comment:
stripped away social link, since it's gonna be a much larger change. 

created issue for tracking: https://github.com/social-plan-it/plan-it-social-web/issues/114